src/main/java/org/sciborgs1155/robot/drive/Drive.java

```diff
@@ -1,263 +1,194 @@
+      leftEncoder.getPosition(), 
+      rightEncoder.getPosition(), 
+      new Pose2d(),
+          VecBuilder.fill(0.05, 0.05, Units.degreesToRadians(5)), //these could probably be adjusted later, taken from wpilib docs
+          VecBuilder.fill(0.5, 0.5, Units.degreesToRadians(30)));
 
-  /** Deadbands and squares inputs */
-  private static double scale(double input) {
-    input = MathUtil.applyDeadband(input, Constants.DEADBAND);
-    return Math.copySign(input * input, input);
-  }
+  /** Creates a new Drive. */
+  public Drive(){
+    leftEncoder.setPositionConversionFactor(CONVERSION);
+    rightEncoder.setPositionConversionFactor(CONVERSION);
+    // Velocity conversion factor should be set to the exact same value as position conversion fact, times 60. NOTE FOR SELF - michael
+    leftEncoder.setVelocityConversionFactor(CONVERSION * 60);
+    leftEncoder.setVelocityConversionFactor(CONVERSION * 60);
 
-  /** Drives the robot based on a {@link DoubleSupplier} for x y and omega velocities */
-  public CommandBase drive(DoubleSupplier vx, DoubleSupplier vy, DoubleSupplier vOmega) {
-    return run(
-        () ->
-            drive(
-                ChassisSpeeds.fromFieldRelativeSpeeds(
-                    xLimiter.calculate(scale(vx.getAsDouble()) * MAX_SPEED * speedMultiplier),
-                    yLimiter.calculate(scale(vy.getAsDouble()) * MAX_SPEED * speedMultiplier),
-                    scale(vOmega.getAsDouble()) * MAX_ANGULAR_SPEED * speedMultiplier,
-                    getHeading())));
-  }
+    rightMotors.setInverted(true);
+    
+    // AutoBuilder.configureLTV()
 
-  /**
-   * Drives the robot based on profided {@link ChassisSpeeds}.
-   *
-   * <p>This method uses {@link Pose2d#log(Pose2d)} to reduce skew.
-   *
-   * @param speeds The desired chassis speeds.
-   */
-  public void drive(ChassisSpeeds speeds) {
-    var target =
-        new Pose2d(
-            speeds.vxMetersPerSecond * Constants.PERIOD,
-            speeds.vyMetersPerSecond * Constants.PERIOD,
-            Rotation2d.fromRadians(speeds.omegaRadiansPerSecond * Constants.PERIOD));
-
-    var twist = new Pose2d().log(target);
-
-    speeds =
-        new ChassisSpeeds(
-            twist.dx / Constants.PERIOD,
-            twist.dy / Constants.PERIOD,
-            twist.dtheta / Constants.PERIOD);
-
-    setModuleStates(kinematics.toSwerveModuleStates(speeds));
+    
   }
 
-  /**
-   * Sets the swerve ModuleStates.
-   *
-   * @param desiredStates The desired ModuleIO states.
-   */
-  public void setModuleStates(SwerveModuleState[] desiredStates) {
-    if (desiredStates.length != modules.size()) {
-      throw new IllegalArgumentException("desiredStates must have the same length as modules");
-    }
-
-    SwerveDriveKinematics.desaturateWheelSpeeds(desiredStates, MAX_SPEED);
-
-    for (int i = 0; i < modules.size(); i++) {
-      modules.get(i).setDesiredState(desiredStates[i]);
-    }
+  public void setVoltage(double voltageR, double voltageL) {
```

Comment:
small style thing but it'd be nice if you could standardize how you notate sides a bit. Right now in some places you add an r or l at the end, like voltageR, in some places you add it at the beginning, like LdriveFF (speaking of which, make sure you're using camelCase! That should be lDriveFF) and in some you write out the whole word like leftEncoder